tests/python/unittest/test_meta_schedule_post_order_apply.py

```diff
@@ -414,5 +617,393 @@ def apply(self, sch: Schedule, block: BlockRV) -> List[Schedule]:
+            "sch.unroll(loop=l28)",
+            "sch.unroll(loop=l31)",
+            "sch.unroll(loop=l32)",
+            "v33, v34 = sch.sample_perfect_tile(loop=l29, n=2, max_innermost_factor=64, decision=[9, 1])",
+            "l35, l36 = sch.split(loop=l29, factors=[v33, v34])",
+            "v37, v38 = sch.sample_perfect_tile(loop=l30, n=2, max_innermost_factor=64, decision=[32, 4])",
+            "l39, l40 = sch.split(loop=l30, factors=[v37, v38])",
+            "sch.reorder(l35, l39, l36, l40, l27, l28, l31, l32)",
+            'b41 = sch.get_block(name="bgemm", func_name="main")',
+            'b42 = sch.cache_write(block=b41, write_buffer_index=0, storage_scope="global")',
+            "l43, l44, l45, l46, l47 = sch.get_loops(block=b41)",
+            "v48, v49, v50, v51 = sch.sample_perfect_tile(loop=l43, n=4, max_innermost_factor=64, decision=[1, 2, 3, 1])",
+            "l52, l53, l54, l55 = sch.split(loop=l43, factors=[v48, v49, v50, v51])",
+            "v56, v57, v58, v59 = sch.sample_perfect_tile(loop=l44, n=4, max_innermost_factor=64, decision=[1, 1, 1, 6])",
+            "l60, l61, l62, l63 = sch.split(loop=l44, factors=[v56, v57, v58, v59])",
+            "v64, v65, v66, v67 = sch.sample_perfect_tile(loop=l45, n=4, max_innermost_factor=64, decision=[1, 1, 1, 9])",
+            "l68, l69, l70, l71 = sch.split(loop=l45, factors=[v64, v65, v66, v67])",
+            "v72, v73, v74, v75 = sch.sample_perfect_tile(loop=l46, n=4, max_innermost_factor=64, decision=[2, 1, 16, 4])",
+            "l76, l77, l78, l79 = sch.split(loop=l46, factors=[v72, v73, v74, v75])",
+            "v80, v81 = sch.sample_perfect_tile(loop=l47, n=2, max_innermost_factor=64, decision=[16, 8])",
+            "l82, l83 = sch.split(loop=l47, factors=[v80, v81])",
+            "sch.reorder(l52, l60, l68, l76, l53, l61, l69, l77, l82, l54, l62, l70, l78, l83, l55, l63, l71, l79)",
+            "sch.reverse_compute_at(block=b42, loop=l77, preserve_unit_loops=True)",
+            'b84 = sch.get_block(name="root", func_name="main")',
+            'sch.annotate(block_or_loop=b84, ann_key="auto_parallel_extent", ann_val=64)',
+            'sch.annotate(block_or_loop=b84, ann_key="auto_vectorize_extent", ann_val=32)',
+            "v85 = sch.sample_categorical(candidates=[0, 16, 64, 512], probs=[0.25, 0.25, 0.25, 0.25], decision=1)",
+            'sch.annotate(block_or_loop=b84, ann_key="auto_unroll_explicit", ann_val=v85)',
+            'b86 = sch.get_block(name="input_tile", func_name="main")',
+            "l87 = sch.sample_compute_location(block=b86, decision=-1)",
+            "sch.compute_at(block=b86, loop=l87, preserve_unit_loops=True)",
+            'b88 = sch.get_block(name="data_pad", func_name="main")',
+            "l89 = sch.sample_compute_location(block=b88, decision=-1)",
+            "sch.compute_at(block=b88, loop=l89, preserve_unit_loops=True)",
+        ],
+    )
+
+
+def test_meta_schedule_post_order_apply_custom_search_space_winograd_cuda():
```

Comment:
Mark this test as "known to fail": https://docs.pytest.org/en/6.2.x/skipping.html#:~:text=An%20xfail%20means%20that%20you,mark.

```suggestion
@pytest.mark.xfail
def test_meta_schedule_post_order_apply_custom_search_space_winograd_cuda():
```